zettarepl/replication/stuck.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding=utf-8 -*-
+import itertools
+import logging
+import time
+
+from zettarepl.transport.interface import ExecException
+
+from .error import StuckReplicationError
+
+logger = logging.getLogger(__name__)
+
+__all__ = ["retry_stuck_replication"]
+
+
+def retry_stuck_replication(func, recoverable_error):
+    for i in itertools.count(1):
+        try:
+            return func()
+        except ExecException as e:
+            if (
+                isinstance(recoverable_error, StuckReplicationError) and
+                "contains partially-complete state" in e.stdout
+            ):
+                logger.warning(
```

Comment:
Am I correct in understanding that this message will be dumped every 60 seconds for 1 hr?

Reply:
@yocalebo actually I reverted my commit, we'll continue logging this every minute, because it is not the only `warning` message that will be occurring every minute with erroneous replication retry.

Reply:
This writes to `zettarepl.log` right?

Reply:
Yes, and to the middleware job log